metadata-utils/src/main/java/com/linkedin/metadata/authorization/PoliciesConfig.java

```diff
@@ -249,6 +249,12 @@ public class PoliciesConfig {
       "Edit Contact Information",
       "The ability to change the contact information such as email & chat handles.");
 
+  // Glossary Node Privileges
+  public static final Privilege MANAGE_CHILDREN_PRIVILEGE = Privilege.of(
```

Comment:
I know we are calling this a "Glossary Node Privilege", but then naming it something really generic (Manage Children). What is the risk of going too generic with this? Would it be better to go more specific with something like "MANAGE_GLOSSARY_CHILDREN"? Disclaimer- I have not deeply thought through it

Reply:
hmm so I think we could certainly reuse this privilege for other things if we start needing to. The risk is that `MANAGE_CHILDREN` could mean slightly different things in different contexts. So in the future - does `MANAGE_CHILDREN` mean the same thing for Glossary Nodes, Domains, and Containers? maybe? maybe not.

so maybe the least risky move here is to make this more specific and call if `MANAGE_GLOSSARY_CHILDREN` so that if we want to have slightly different meanings for different parent contexts.

It would also give more flexibility when creating policies to be more fine-grained - maybe you want to create a policy where users can manage children of nodes they create, but only edit domains they own, but not necessarily their children - they could then add the `MANAGE_GLOSSARY_CHILDREN` privilege but omit the `MANAGE_DOMAIN_CHILDREN` privilege.

The bummer then is if we start having a lot of these, it feels repetitive. However, I don't really see that happening. Also we're pretty repetitive in our privileges per entity type anyways, this kind of fits in with the current approach of giving very fine-grained control over things.

After talking this out I'm going to go ahead and change the name to be more specific! However if you feel different let me know and I'll be happy to discuss.